emily/handler/src/database/accessors.rs

```diff
@@ -696,13 +696,175 @@ pub async fn set_api_state(context: &EmilyContext, api_state: &ApiStateEntry) ->
 
 // Limits ----------------------------------------------------------------------
 
+/// Returns height of first stacks block ancored to bitcoin block with provided height
+async fn calculate_associated_stacks_block_fast(
```

Comment:
I think that we should implement this using one simple query to DynamoDB where we just get the least stacks height with a given bitcoin anchor height. That query will just be easier to understand that what is here, even if it's a little more expensive.